.coderabbit.yaml

```
@@ -0,0 +1,205 @@
+        - Use `currentColor` for fill, stroke, and color attributes
+        - Remove all namespace attributes (e.g. `xmlns:*`)
+    - path: "packages/react/src/components/**/*.test.tsx"
+      instructions: |
+        Review the component tests against the following rules:
+        - Test all variant prop values (e.g. `primary`, `secondary`, `default`)
+        - Test all interactive states (e.g. focus, expanded, disabled)
+        - Test layout-related behaviors (e.g. responsiveness, media queries, overflow)
+        - Test all props, including event handlers and refs
+        - Run axe tests for all variants and states
+        - Follow React Testing Library best practices:
+          - Use `screen` instead of `render`
+          - Use role-based queries first, then text-based queries, and finally test-id queries as a last resort
+          - Use query* only for checking non-existence
+          - Use userEvent for event simulation
+          - Use find* instead of waitFor
+          - Use explicit assertions with get* variants
+        - Avoid abbreviations in variable and function names
+    - path: "docs/pages/components/**/*.mdx"
+      instructions: |
+        Review the component documentation against the following rules:
+        - Include a "Related Components" section when applicable
+        - Represent single types as strings without arrays (e.g. use `'ReactNode'` not `['ReactNode']`)
+        - Document all component props with name, type, and description
+        - Avoid redundant type declarations
+        - Maintain consistent type declarations across properties
+        - Ensure documentation examples match implementation
+        - Verify correct spelling in prop text
+        - Use consistent heading case (e.g. "Multiselect Disabled" not "Multiselect disabled")
+
+  abort_on_close: true
+  disable_cache: false
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    ignore_title_keywords: []
+    labels: []
+    drafts: true
+    base_branches: []
+  finishing_touches:
+    docstrings:
+      enabled: true
+  tools:
+    shellcheck:
+      enabled: true
+    ruff:
+      enabled: false
+    markdownlint:
+      enabled: true
+    github-checks:
+      enabled: true
+      timeout_ms: 90000
+    languagetool:
+      enabled: true
+      enabled_rules: []
+      disabled_rules: []
+      enabled_categories: []
+      disabled_categories: []
+      enabled_only: false
+      level: default
+    biome:
+      enabled: false
+    hadolint:
+      enabled: false
+    swiftlint:
+      enabled: false
+    phpstan:
+      enabled: false
+      level: default
+    golangci-lint:
+      enabled: false
+    yamllint:
+      enabled: true
+    gitleaks:
+      enabled: false
+    checkov:
+      enabled: true
+    detekt:
+      enabled: false
+    eslint:
+      enabled: false
+    rubocop:
+      enabled: false
+    buf:
+      enabled: false
+    regal:
+      enabled: false
+    actionlint:
+      enabled: true
```

I think GitHub already does this for us, no?

```suggestion
    actionlint:
      enabled: false
```

---

I don't think it's needed. I also don't know if it hurts, though. I think it should just provide helpful information to CodeRabbit. I can disable, though, if you'd prefer

---

My preference being use native tools (where available) vs this.